Docker_exercises/ex 6/frontend/my-react-app/src/components/Home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Header = () => {
```

Comment:
This is a Home Component why you have a Header Component code here?
And where are you making a network request to the express server to fetch the data?